enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -160,7 +159,7 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
           }
         } else {
           // arguably shouldn't log anything on success
-          logger.info("No " + foundError + " found");
+          logger.info("No" + foundError + " found");
```

Comment:
seems like there should be a space after No here.

Reply:
Fixed
(The space was in `foundError` but it was confusing.)